app/layout.tsx

```diff
@@ -32,14 +33,16 @@ export const metadata = {
   }
 }
 
+export const revalidate = 86400;
```

Comment:
Can you tell me more about this revalidation time? Why only one day here but 30 on the page route? Wouldn't you want the layout to be cached longer than the slots?

Reply:
This allows the menu to revalidate more frequent since the page invalidation doesn't invalidate that.